Make a special welcome message for Shipping/Service prep.

DRracer opened the pull request:
Another request from our Service dept. - the user shall be prevented from skipping the intro wizard, because otherwise some preset/calibrated features will look like not done - especially live-z calibration.

And since there are users, who send a machine to service to perform 1st layer calibration only, they must not omit the Z-calibration at the start after shipping.

PFW-1161

Commit:
Make a special welcome message for shipping/service prep

Another request from our Service dept. - the user shall be prevented
from skipping the intro wizard, because otherwise some preset/calibrated
features will look like not done - especially live-z calibration.

And since there are users, who send a machine to service to perform 1st
layer calibration only, they must not omit the Z-calibration at the
start after shipping.

Commit:
Keep wizard flag==2 even when a user restarts during Z-calibration

i.e. prevent jumping into the standard wizard if reset during Z-
calibration

Commit:
Update translations of the Shipping/Service prep welcome msg

3d-gussner commented on Firmware/ultralcd.cpp:
```diff
@@ -4986,14 +4986,18 @@ void lcd_wizard(WizState state)
-			wizard_event = lcd_show_multiscreen_message_yes_no_and_wait_P(_i("Hi, I am your Original Prusa i3 printer. Would you like me to guide you through the setup process?"), false, true);////MSG_WIZARD_WELCOME c=20 r=7
-			if (wizard_event) {
+			if( eeprom_read_byte((uint8_t*)EEPROM_WIZARD_ACTIVE)==2){
+				lcd_show_fullscreen_message_and_wait_P(_i("Hi, I am your Original Prusa i3 printer. I will guide you through a short setup process, in which the Z-axis will be calibrated. Then, you will be ready to print."));////MSG_WIZARD_WELCOME_SHIPPING c=20 r=7
```
Put the message in `messages.*` as MSG_WIZARD_WELCOME_SHIPPING c=20 r=7 ?
Did you run https://github.com/prusa3d/Prusa-Firmware/pull/2665 to check if the translated messages fit in 7 rows?

DRracer replied:
good point, thanks

3d-gussner commented on Firmware/ultralcd.cpp:
```diff
@@ -4986,14 +4986,18 @@ void lcd_wizard(WizState state)
-				eeprom_update_byte((uint8_t*)EEPROM_WIZARD_ACTIVE, 0);
-				end = true;
+			} else {
+				wizard_event = lcd_show_multiscreen_message_yes_no_and_wait_P(_i("Hi, I am your Original Prusa i3 printer. Would you like me to guide you through the setup process?"), false, true);////MSG_WIZARD_WELCOME c=20 r=7
```
Same as above?

3d-gussner commented on lang/lang_en.txt:
```diff
@@ -142,6 +142,9 @@
 "Card removed"
 
+#
+"Checking file"
```
Like it that you check `not_tran_*.txt` files.

3d-gussner commented on lang/lang_en.txt:
```diff
@@ -871,6 +877,9 @@
 "Unload"
 
+#
+"Sheet %.7s\x0aZ offset: %+1.3f mm\x0a%cContinue\x0a%cStart from zero"
```
Adding this to lang files will generate issues in https://github.com/prusa3d/Prusa-Firmware/pull/2665

DRracer replied:
I've already forgotten if there was a reason for not including this translation

3d-gussner commented on lang/lang_en.txt:
```diff
@@ -382,6 +385,9 @@
 #MSG_LAST_PRINT_FAILURES c=20
 "Last print failures"
 
+#
```
should be at least `# c=20 r=7` better `#MSG_WIZARD_WELCOME_SHIPPING c=20 r=7` if moved to `messages.*`

3d-gussner reviewed (commented):
Some comments on messages and translations

Commit:
Move wizard msgs into messages.h/c + add more translation markers

Commit:
Remove unrelated (but still missing) translations

This needs to be fixed in 3.10 final. Rumors say some of the translation
scripts even fail on a message containing both % and a backslash

Commit:
Remove unrelated translations from master lang_en.txt as well
